### PR TITLE
update documentation and remove Logitech documents

### PR DESCRIPTION
**Pull request by pfps**
Addresses #591 

Added information on how to add a new receiver and a new device.
Updated issue templates

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Feature                                | ID       | Status             | Notes
 `DEVICE_RESET`                         | `0x1802` | :x:                |
 `OOBSTATE`                             | `0x1805` | :x:                |
 `CONFIG_DEVICE_PROPS`                  | `0x1806` | :x:                |
-`CHANGE_HOST`                          | `0x1814` | :x:                |
+`CHANGE_HOST`                          | `0x1814` | :x:                | implementation in progress
```
Comment (FFY00) [inline docs/features.md:36]: I think we can use one of the following emoji here: :hammer: (`:hammer:`), :wrench: (`:wrench:`).

Comment (pfps) [reply docs/features.md]: Works for me. :-)

Comment (FFY00) [reply docs/features.md]: No, I meant in the emoji column, instead of `:x:` :joy:

Comment (pfps) [reply docs/features.md]: Oh, well, probably good enough for now.  I

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ assignees: ''
 ---
 
 **Information**
-<!-- Do not bother opening an issue for version 0.9.2 or older, it is extremely outdated. Please update to the latest one and see if your issue persists. -->
-- Solaar version (`git describe --tags` if cloned from Solaar repository):
+<!-- Do not bother opening an issue for version 1.0 or older.  Please update to the latest version and see if your issue persists. -->
```
Comment (Klap-in) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: 1.0 sounds for me a bit overlapping with 1.0.2. So I guess it is better to be more explicit. 
What about: `for version 1.0.1 or older.` ?

Comment (pfps) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: I can see that. How about "older than version 1.0.1"?

Comment (Klap-in) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: Fine for me

**Pull request merged**